src/Ilios/CoreBundle/Entity/LearningMaterial.php

```diff
@@ -296,13 +293,12 @@ public function getToken()
      */
     public function generateToken()
     {
-        $generator = new SecureRandom();
-        $random = $generator->nextBytes(128);
-        
+        $random = random_bytes(128);
```

Comment:
Ok looks like symfony/security includes symfony PHP 7 polyfill which includes this dependancy.  I guess we're not likely to remove symfony security, so we should be good to go here.

Nothing to see here.... move along.